### ISSUE TITLE
Intercom framework does not support bitcode

### ISSUE THREAD

**Issue by renssies** [intercom/intercom-ios#90]
iOS 9 is bringing bitcode to rewrite apps on Apple's servers allowing for smaller app binaries or other changes. 
However with bitcode on, you can't compile your app because intercom does not support it. 
Probably because iOS 9 isn't officially out yet. Please enable bitcode for intercom as soon as possible, maybe on a separate branch so we can submit apps with iOS 9 and bitcode support starting today (when the iOS 9 GM launches).

The full error: 
```d: '/Users/[REDACTED]/Pods/Intercom/Intercom.framework/Intercom(AFHTTPRequestOperation.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)```

**Comment by danielmackey** [comment]
I have also run into this using iOS9 and Xcode 7.0. Both the cocoapods and manual installation processes fail during app build.

**Comment by jtreanor** [comment]
Thanks @renssies and @danielmackey 

We're are currently looking into this. I will keep you updated here as we know more.

**Comment by martingsaunders** [comment]
Is there an ETA on bit odd support. We have a WatchOS2 app to submit this week and will have to remove Intercom if bitcode isn't ready.

**Comment by martingsaunders** [comment]
Or bitcode even...

**Comment by jtreanor** [comment]
@martingsaunders soon. It will either be this week or early next week. We are ironing out the iOS 9 kinks at the moment.

**Comment by martingsaunders** [comment]
Ok, great stuff. I'll hang on in that case. 👍🏻

**Comment by roland9** [comment]
@martingsaunders you might be aware of this, just mentioning it here:  
since Apple went GM with Xcode 7, you can indeed build 'mixed' apps: you can have Bitcode *disabled* in the iOS app, but *enabled* in the watchOS extension (iTunes Connect won't accept WatchKit extensions without Bitcode). 
-> That wasn't allowed during the beta period, but now it's perfectly fine (as they announced during WWDC if I remember correctly)

So in other words: you can have Intercom (and other 3rd party libraries) in your iOS app when you submit your Watch app for App Store review.

**Comment by ded** [comment]
argh. Intercom is going to be an essential part of our app, and with iOS9 officially out today (XCode went GM just a few days ago), we have no way to build an archive :disappointed: 

At this point I would presume this must be the highest priority at the moment — and it is indeed a blocker for us.

**Comment by jtreanor** [comment]
Thanks @ded. Yes, this is the highest priority right now.

You should be able to build an archive for now by turning off Bitcode.

**Comment by ded** [comment]
Thanks @jtreanor — I was able to build the archive, but it was an invalid binary (not enabling us to submit to App Store). It seems to be [happening to others](http://stackoverflow.com/questions/32526526/invalid-swift-support-files-don-t-match), however that is likely an unrelated problem to Intercom.

**Comment by jtreanor** [comment]
Sorry to hear that @ded. I hope Apple resolve it soon.

The new version will be out early next week.

**Comment by roland9** [comment]
@ded I had the same issue (invalid binary) when I tried to submit with Xcode 7 GM.

The only solution I found was to revert to a stage where I still used Xcode 6.4 (before I made any changes with Xcode 7 beta in my project), and then import all the changes, files since then. 
Then it was fine; was able to submit to the App Store no issues. 

Painful, but my conclusion was that Xcode beta 'broke' the project somehow in the process, and GM couldn't recover.

**Comment by ded** [comment]
@roland9 following the instructions from that link to copy over the files fixed the issue (I still received a warning after the Organizer finished (saying [API analysis is too large](http://cl.ly/image/0s0N0s441s0X))

**Comment by roland9** [comment]
I got the same - but because it was just a warning I ignored it
And the submission was fine :)
Good luck!

**Comment by jtreanor** [comment]
Version 2.3.5 has been released with iOS 9 and Bitcode support.

Thanks everyone for all your feedback on this :smile:

**Comment by mfkp** [comment]
@jtreanor - FYI 2.3.5 breaks Xcode 6 compatibility. Seems related to this bug: http://www.openradar.me/21884601

I get hundreds of the duplicate symbol errors, such as the referenced
```duplicate symbol _llvm.cmdline:```

I downgraded to 2.3.4 and it's verified working in Xcode 6.4

**Comment by jtreanor** [comment]
Thanks @mfkp. I have opened a separate issue to track this here: #98

**Comment by af-inf** [comment]
@jtreanor I could not "prepare an archive for export" because I had the "bitcode" checkbox on. This is with latest intercom on latest react-native.

**Comment by jtreanor** [comment]
@af-inf Could you let me know what error you are seeing for this?
Intercom should be fully compatible with Bitcode.
